src/App.tsx

```diff
@@ -1,24 +1,173 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect, useState, useCallback, FormEvent,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+export const USER_ID = 9944;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<TodoStatus>(TodoStatus.All);
+  const [isError, setIsError] = useState(false);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NoError);
```

Comment:
`isError` variable can be calculated based on `errorType`
```suggestion
  const [newTodoTitle, setNewTodoTitle] = useState('');
  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.NoError);
  const isError = errorType !== ErrorType.NoError;
```